server/manager/src/main/java/org/apache/accumulo/manager/compaction/queue/CompactionJobPriorityQueue.java

```diff
@@ -136,18 +141,38 @@ public synchronized boolean add(TabletMetadata tabletMetadata, Collection<Compac
     return true;
   }
 
+  public long getMaxSize() {
+    return maxSize;
+  }
+
+  public long getRejectedJobs() {
+    return rejectedJobs.get();
+  }
+
+  public long getDequeuedJobs() {
+    return dequeuedJobs.get();
+  }
+
+  public long getQueuedJobs() {
+    return tabletJobs.size();
+  }
+
+  public long getLowestPriority() {
+    return jobQueue.lastKey().job.getPriority();
+  }
```

Comment:
`tabletJobs.size()` will return the number of tablets that have jobs in the queue and a tablet could have multiple, so I think `jobQueue..size()` is better.  Need to syncronize when accessing the jobQueue data struct as its not a concurrent data struct.

```suggestion
  public synchronized long getQueuedJobs() {
    return jobQueue.size();
  }

  public synchronized long getLowestPriority() {
    return jobQueue.lastKey().job.getPriority();
  }
```